commands/bridge_configure.go

```diff
@@ -90,12 +105,38 @@ func promptName() (string, error) {
 }
 
 var bridgeConfigureCmd = &cobra.Command{
-	Use:     "configure",
-	Short:   "Configure a new bridge.",
+	Use:   "configure",
+	Short: "Configure a new bridge.",
+	Long: `	Configure a new bridge by passing flags or/and using interactive terminal prompts. You can avoid all the terminal prompts by passing all the necessary flags to configure your bridge.
+	Repository configuration can be made by passing or the --url flag or the --project and/or --owner flags. If the three flags are provided git-bug will use --project and --owner flags.
```

Comment:
```suggestion
	Repository configuration can be made by passing either the --url flag or the --project and --owner flags. If the three flags are provided git-bug will use --project and --owner flags.
```
Both --project and --owner are required if --url is not given, correct ?